public/video-ui/src/services/pandaReqwest.js

```diff
@@ -9,8 +9,8 @@ function poll(reqwestBody, timeout) {
   function makeRequest(resolve, reject) {
     reqwest(reqwestBody)
       .then(response => resolve(response))
-      .fail(err => {
-        if (err === 419) {
+      .catch(err => {
```

Comment:
The `Promise` at the bottom is `.catch` because it's a plain JS `Promise`. This is why we used `Q` in tagmanager, because it presents a consistent API for handling promises with `Reqwest`